right/src/usb_commands/usb_command_get_device_state.c

```diff
@@ -22,5 +34,6 @@ void UsbCommand_GetKeyboardState(void)
     SetUsbTxBufferUint8(4, MODULE_CONNECTION_STATE(UhkModuleDriverId_LeftModule));
     SetUsbTxBufferUint8(5, MODULE_CONNECTION_STATE(UhkModuleDriverId_RightModule));
     SetUsbTxBufferUint8(6, ActiveLayer | (ActiveLayer != LayerId_Base && !ActiveLayerHeld ? (1 << 7) : 0) ); //Active layer + most significant bit if layer is toggled
+    SetUsbTxBufferUint8(7, getErrorState());
```

Comment:
`IsEepromBusy` or `MERGE_SENSOR_IS_MERGED` should be composed with `getErrorState()` instead because the device protocol will be optimized later to allow for multiple commands per packet, so commands should be as short as possible.

I'm unsure whether errors and warnings will be extended with additional types depending on which the error state should be shifted with 2 or more bits when composed.